app/components/course-page/course-completed-card.ts

```diff
@@ -0,0 +1,30 @@
+import { tracked } from '@glimmer/tracking';
+import Component from '@glimmer/component';
+import congratulationsImage from '/assets/images/icons/congratulations.png';
+import type RepositoryModel from 'codecrafters-frontend/models/repository';
+
+interface Signature {
+  Element: HTMLDivElement;
+
+  Args: {
+    repository: RepositoryModel;
+  };
+}
+
+export default class CourseCompletedCardComponent extends Component<Signature> {
+  congratulationsImage = congratulationsImage;
+
+  @tracked configureGithubIntegrationModalIsOpen = false;
+
+  get shouldRenderDefaultCompletionMessage() {
```

Comment:
@ryan-gang this can be placed directly in the template as `{{if @repository.course.completionMessageMarkdown}}`, indirection probably isn't worth-it